lib/task/digitalobject/digitalObjectLoadTask.class.php

```diff
@@ -96,30 +102,32 @@ public function execute($arguments = array(), $options = array())
 
     $this->disableNestedSetUpdating = ($options['skip-nested-set-build']) ? true : false;
 
-    $this->logSection('digital-object', "Load digital objects from {$arguments['filename']}...");
+    $this->logSection('digital-object', ($options['replace'] ? "Replace" : "Load") . " digital objects from {$arguments['filename']}...");
```

Comment:
Nitpick: might be cleaner to use sprintf, something like the below, rather than string concatenation but either way works.
```
$operation = $options['replace'] ? "Replace" : "Load";
$this->logSection('digital-object', sprintf('%s digital objects from %s...', $operation, $arguments['filename']));
```

Reply:
Change completed. Good idea - definitely cleaner!